apps/web/src/styles/interaction.css

```
@@ -23,101 +23,101 @@
 
   .interaction-brand-normal-inverse {
-    @apply focus-visible:shadow-focus-visible relative outline-none before:absolute before:inset-0 before:rounded-[inherit] hover:before:bg-[rgba(74,74,214,0.08)] active:before:bg-[rgba(74,74,214,0.08)];
+    @apply relative outline-none before:absolute before:inset-0 before:rounded-[inherit] hover:before:bg-[rgba(74,74,214,0.08)] focus-visible:shadow-focus-visible active:before:bg-[rgba(74,74,214,0.08)];
   }
 
   .interaction-brand-subtle-inverse {
-    @apply focus-visible:shadow-focus-visible relative outline-none before:absolute before:inset-0 before:rounded-[inherit] hover:before:bg-[rgba(100,101,241,0.05)] active:before:bg-[rgba(100,101,241,0.05)];
+    @apply relative outline-none before:absolute before:inset-0 before:rounded-[inherit] hover:before:bg-[rgba(100,101,241,0.05)] focus-visible:shadow-focus-visible active:before:bg-[rgba(100,101,241,0.05)];
   }
 
   /* scale inversed false*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)